PSCompatibilityAnalyzer/Microsoft.PowerShell.CrossCompatibility/Query/Modules/ModuleData.cs

```diff
@@ -66,16 +72,33 @@ public ModuleData(string name, Version version, Data.Modules.ModuleData moduleDa
         /// <summary>
         /// Aliases exported by the module.
         /// </summary>
-        public IReadOnlyDictionary<string, CommandData> Aliases => _commands.Value.Item3;
+        public IReadOnlyDictionary<string, IReadOnlyList<CommandData>> Aliases => _lazyAliases.Value;
+
+        private IReadOnlyDictionary<string, IReadOnlyList<CommandData>> CreateAliasTable(IReadOnlyDictionary<string, string> aliases)
+        {
+            if (aliases == null || aliases.Count == 0)
+            {
+                return null;
+            }
+
+            var aliasTable = new Dictionary<string, IReadOnlyList<CommandData>>();
```

Comment:
Should we rather create a case insensitive dictionary by adding `StringComparer.OrdinalIgnoreCase` to the constructor?

Reply:
Good catch